src/tsconfig.spec.json

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
You moved the config here to the main tsconfig ? I am concern about the jasmine and node types definition below. Not having them means the jasmine terms will not be recognized.